src/main/java/com/eprosima/fastdds/idl/parser/typecode/StringTypeCode.java

```diff
@@ -14,9 +14,12 @@
 
 package com.eprosima.fastdds.idl.parser.typecode;
 
+import com.eprosima.fastcdr.idl.context.Context;
 import com.eprosima.idl.parser.exception.RuntimeGenerationException;
 import com.eprosima.idl.parser.typecode.Kind;
 
+import org.stringtemplate.v4.ST;
```

Comment:
Is this import really required? I do not think so

Reply:
Done in 020947b